rcongui/src/components/PlayerView/playerActions.js

```diff
@@ -128,7 +128,7 @@ class ReasonDialog extends React.Component {
                 label="Reason"
                 variant="outlined"
                 margin="dense"
-                helperText="The message that will be displayed to the player. A message is mandatory"
+                helperText="The message that will be displayed when the `Watchlist` is triggered. A message is mandatory."
```

Comment:
That's not right because this component is used for all the different actions. So this will be displayed for pusnih/kick/ban as well. You could make a condition based on the `{open.actionType}`

Reply:
Yeah i thought it might be the case when reading the code section.
Just based on the current text i would assume the player is notified.